docs/setting_up_your_development_environment.md

```diff
@@ -47,14 +45,25 @@ You'll need:
    sudo apt install git make libpq-dev
    sudo snap install --classic node
    ```
-2. Follow [Docker's install instructions](https://docs.docker.com/get-docker/) including the [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/). You **don't** need to install Docker Compose
-3. Follow [pyenv's installation instructions](https://github.com/pyenv/pyenv#installation):
+2. Follow [GitHub CLI's install instructions](https://cli.github.com/)
+3. Follow [Docker's install instructions](https://docs.docker.com/get-docker/) including the [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/). You **don't** need to install Docker Compose
+4. Follow [pyenv's installation instructions](https://github.com/pyenv/pyenv#installation):
    1. The [Basic GitHub Checkout](https://github.com/pyenv/pyenv#basic-github-checkout) method works best on Ubuntu
    2. [Set up your shell](https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv) for pyenv
    3. [Install the Python build dependencies](https://github.com/pyenv/pyenv/wiki#suggested-build-environment)
       that pyenv needs
 </details>
 
+Set up Git &rarr; GitHub.com authentication
+-------------------------------------------
+
+Set up Git to authenticate requests to GitHub.com using your GitHub.com account.
+The easiest way to do this is to run GitHub CLI's [`gh auth login` command](https://cli.github.com/manual/gh_auth_login) and answer **Yes** (the default answer) when it asks **Authenticate Git with your GitHub credentials?**
+
+```terminal
+$ gh auth login -p https -h github.com
+```
```

Comment:
This is the easiest way to set up Git->GitHub.com authentication and the first way mentioned in GitHub's docs. Much easier than setting up an SSH key.